strapi/strapi
# [DEITS] Import assets

### What does it do?

It allows importing assets from a folder to a strapi instance

### Why is it needed?

To import exported assets from a file

### How to test it?

You can run the tests in the local-file-source-provider and in the local-strapi-destination-provider to check that all pass and you can run the command `yarn strapi import <name_of_exported_file>` and check that the assets are imported correctly
### Related issue(s)/PR(s)

Let us know if this is related to any issue/pull request
